pypit/arload.py

```diff
@@ -316,18 +316,7 @@ def load_master(name, exten=0, frametype='<None>'):
     frame : ndarray
       The data from the master calibration frame
     """
-    if frametype is None:
-        msgs.info("Loading a pre-existing master calibration frame")
-        try:
-            hdu = pyfits.open(name)
-        except IOError:
-            msgs.error("Master calibration file does not exist:"+msgs.newline()+name)
-        msgs.info("Master {0:s} frame loaded successfully:".format(hdu[0].header['FRAMETYP'])+msgs.newline()+name)
-        head = hdu[0].header
-        data = hdu[exten].data.astype(np.float)
-        return data, head
-        #return np.array(infile[0].data, dtype=np.float)
-    else:
+    if frametype == 'wv_calib':
         from linetools import utils as ltu
         msgs.info("Loading Master {0:s} frame:".format(frametype)+msgs.newline()+name)
         if frametype == 'wv_calib':
```

Comment:
I feel like this is no longer necessary, right? It's the same if statement as the one 3 lines above.

Also looks like the following (old) else statement on line 325 has been moved to line 340.